modules/javafx.media/src/main/native/gstreamer/gstreamer-lite/gst-plugins-good/sys/directsound/gstdirectsoundnotify.cpp

```diff
@@ -51,6 +51,12 @@ void ReleaseNotificator(void *pObject) {
 bool GSTDirectSoundNotify::Init(GSTDSNotfierCallback pCallback, void *pData) {
   m_pCallback = pCallback;
   m_pData = pData;
+  bool bCallCoUninitialize = true;
+  bool bResult = false;
+
+  if (FAILED(CoInitialize(NULL))) {
```

Comment:
As per the [`CoUninitialize` doc](https://docs.microsoft.com/en-us/windows/win32/api/combaseapi/nf-combaseapi-couninitialize#remarks), `CoUninitialize` should also be called if `CoInitialize` returns S_FALSE. 
Can you please check.

Reply:
It will be called. S_FALSE is not a failure and defined as 1. FAILED macro returns true if error code < 0.